src/Misc/layoutbin/en-US/strings.json

```diff
@@ -58,6 +58,165 @@
+    "./config.sh --unattended  --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --work usr/local/agent_work",
+    "",
+    "Run as a Windows service that logs on as NetworkService (Windows only)",
+    ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsService",
+    "",
+    "Run as a Windows service that logs on as a domain account (Windows only)",
+    ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsService --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword",
+    "",
+    "Run as an auto logon agent (Windows only)",
+    ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsAutoLogon --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword",
+    "",
+    "Run as an auto logon agent and do not restart after configuration (Windows only)",
+    ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsAutoLogon --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword --noRestart",
+    "",
+    "Deployment group agent run as Windows service that logs on as Local System (Windows only)",
+    ".\\config.cmd --unattended --deploymentGroup --url https://myaccount.visualstudio.com --auth pat --token myToken --projectName myProject --deploymentGroupName myDeploymentGroup --agent myAgent --addDeploymentGroupTags --deploymentGroupTags \"tag1,tag2\" --runAsService --windowsLogonAccount \"NT AUTHORITY\\SYSTEM\"",
+    "",
+    "Deployment group agent with tags",
+    ".{0}config.{1} --unattended --deploymentGroup --url https://myaccount.visualstudio.com --auth pat --token myToken --projectName myProject --deploymentGroupName myDeploymentGroup --agent myAgent --addDeploymentGroupTags --deploymentGroupTags \"tag1,tag2\" --runAsService --windowsLogonAccount \"NT AUTHORITY\\SYSTEM\" --acceptTeeEula"
+  ],
+  "CommandLineHelp_Env": [
+    "CLI-WIDTH-TOTAL-(105-CHARS)------------------------------------------------------------------------------",
+    "Environment variables:",
+    "Any command line argument can be specified as an environment variable. Use the format",
+    "VSTS_AGENT_INPUT_<ARGUMENT_NAME>. For example: VSTS_AGENT_INPUT_PASSWORD"
+  ],
+  "CommandLineHelp_LocalRun": [
+    "CLI-WIDTH-TOTAL-(105-CHARS)------------------------------------------------------------------------------",
+    ".{0}run.{1} localRun [options]",
+    "",
+    "This command is currently in preview. The interface and behavior will change in a future version.",
+    "",
+    "Runs a job in local mode. Specify --url to update the local task cache if newer versions are available",
+    "for any referenced tasks.",
+    "",
+    "{2}",
```

Comment:
`{2}` is replaced with the common options. I think for local run it makes sense for the common options to be after `Run options:` (for config and remove it still makes sense for the common options to be first)